src/bindable-property.js

```diff
@@ -294,16 +298,25 @@ export class BindableProperty {
 }
 
 /**
+ * @param instance the view model instance
+ * @param propertyName name of property changed
+ * @param newValue
+ * @param oldValue
+ */
+function callRefelection(instance: Object, propertyName: string, newValue, oldValue) {
+  let { __element__, __reflections__ } = instance.__observers__;
+  if (!__element__) return;
+  __reflections__[propertyName].call(instance, __element__, propertyName, newVal, oldVal);
+}
+
+/**
  * @private
  * Used for avoid creating mapping function multiple times
- * @param {Element} element 
- * @param {string} propertyName 
- * @param {any} newValue 
  */
-function propToAttr(element, propertyName, newValue) {
-  if (newValue == null) {
-    element.removeAttribute(propertyName);
+function propToAttr(element: Element, propertyName: string, newValue: any) {
+  if (newValue === null || newValue === void 0) {
```

Comment:
OK I might be nitpicking but I prefer `newValue === undefined`, for clarity.
Minifiers will do the job of using the smallest code.

I know users _could_ redefine `undefined` but a lot of other parts won't work if they do.

Reply:
I was trying to do minifier job, will avoid next PRs 👍